BDArmory/UI/BDArmoryAIGUI.cs

```diff
@@ -437,7 +437,8 @@ void SetInputFields(Type AIType)
                     inputFields["minAltitude"].maxValue = ActivePilot.UpToEleven ? 60000 : 6000;
                     inputFields["maxAltitude"].maxValue = ActivePilot.UpToEleven ? 100000 : 15000;
 
-                    inputFields["maxSpeed"].maxValue = ActivePilot.UpToEleven ? 3000 : 800;
+                    if (BDArmorySettings.RUNWAY_PROJECT && BDArmorySettings.RUNWAY_PROJECT_ROUND == 55) inputFields["maxSpeed"].maxValue = ActivePilot.UpToEleven ? 600 : 600;
```

Comment:
`ActivePilot.UpToEleven ? 600 : 600` is a bit redundant.

Reply:
so it is.